### ISSUE TITLE
Packages

### ISSUE THREAD

**Issue by fst3a** [ZeroNoFun/CheatKit#15]
.deb and .rpm + others auto packaging would be nice to have.

**Comment by fst3a** [comment]
Depends on #14 and #16

**Comment by fst3a** [comment]
Partly solved in d7c8ff7e8f0bf313f0415d6e74f951341e9fcf1d (.rpm provided in tag/1.3).
